clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -157,14 +193,27 @@ object UncommittedGarbageCollector {
       firstSlice: String,
       startTime: java.time.Instant,
       success: Boolean,
+      expiredAddresses: DataFrame,
       removed: DataFrame
   ): Unit = {
-    val reportDst = s"${storageNamespace}_lakefs/retention/gc/uncommitted/$runID"
+    val reportDst = reportPath(storageNamespace, runID)
     writeJsonSummary(reportDst, runID, firstSlice, startTime, success, removed.count())
 
+    expiredAddresses.write.parquet(s"$reportDst/marked")
     removed.write.parquet(s"$reportDst/deleted")
   }
 
+  private def reportPath(storageNamespace: String, runID: String): String = {
+    val report = s"${storageNamespace}_lakefs/retention/gc/uncommitted/$runID"
+    report
```

Comment:
Do I know Scala and this may work?
```suggestion
     s"${storageNamespace}_lakefs/retention/gc/uncommitted/$runID"
```

Reply:
done